### DISCUSSION TITLE
Nice new keyboard -- Nuphy Air60

### DISCUSSION THREAD

**Discussion by bwhitman** [shorepine/tulipcc#13] [Show and tell]
I picked up this 60% keyboard for my personal Tulip and love it. It's narrow and shallow but with a good layout for me (I love having an inverted T arrow key setup). Works great on Tulip. I do want to figure out how to address / control  the HID RGB LEDs though! Probably not too hard to figure out. [It's a Nuphy Air60](https://nuphy.com/collections/keyboards/products/air60). 

![IMG_0518](https://user-images.githubusercontent.com/76612/220956453-bd19aa8e-4bc0-4311-b074-4fa3e757ba52.jpeg)

**Comment by bwhitman** [comment]
(You can also buy it with a "folio" -- a folding case that converts into a stand presumed for an iPad with the keyboard butting up against it. It fits the 10.1" HOTMCU display great, if you were nervous about building your own case.)

**Comment by japuzzo** [comment]
This is the keyboard I was going to test with once I get to the hardware stage:

![image](https://user-images.githubusercontent.com/1309858/221317281-eb40d118-2f15-4e37-9752-b8ac3d9474cd.png)

https://www.amazon.com/Mechanical-Keyboard-MageGee-Switches-Portable/dp/B09M6GSXBJ

**Reply by bwhitman** [reply]
ooh, nice one! I wonder if there's a standard for controlling these backlit LEDs
